### PR TITLE
NTR 'contains measured amount'

### PR DESCRIPTION
**Pull request by jmwhorton**
Addresses new term request: #712 (original label was 'has actual load')

Please note:  imports were added manually to other_import.owl.  I couldn't see another place that made more sense.  If I have added the imports incorrectly, please let me know and I'm happy to make the change.

### CODE REVIEW

**Comment by wdduncan** [comment]
@jmwhorton please adress the following:

1. We can discuss at the next meeting, but I would prefer the label to more explicitly reflect that the range is a measurement data. E.g., "container amount measured by". "contains amount" can still be a synonym (e.g., [has_exact_synonym](http://www.geneontology.org/formats/oboInOwl#hasExactSynonym).

2. Change 
```
A relation that relates a container to a measurement datum that specifies the actual amount of material in the container.
```
to
```
A relation between a container and measurement datum that specifies the actual amount of material in the container.
```

3. Instead of `term editor` (IAO_0000117), use annotation `dcterms:contributor` and the editor's ORCID. Each contributor gets their own contribution annotation.  
As an example see [positively correlated with](http://purl.obolibrary.org/obo/RO_0017003). (You will have to open in Protege. OLS doesn't pull up the term.)

4. Can you please provide an [example of usage](http://purl.obolibrary.org/obo/IAO_0000112).

5. The [definition source](https://ontobee.org/ontology/IAO?iri=http://purl.obolibrary.org/obo/IAO_0000119) annotation doesn't seem relevant here. "Contains" is a common word and the contributors are listed if anyone has questions.

6. Is there an inverse relation? E.g., "measures container amount"

**Comment by wdduncan** [comment]
@nataled I see your point.

> "this beaker 'container amount measured by' Mettler scale".

I made the suggestion b/c I was trying to make it more clear that the range is a `measurement datum` and not a literal.

**Comment by nataled** [comment]
@wdduncan substituting 'as' in place of 'by' might come a bit closer (in my mind, at least) in evoking the correct range type. Even so, the 'as' version could just as likely have someone think that 'weight' or 'volume' would be an appropriate response. I'm not sure how best to do it.

**Comment by jmwhorton** [comment]
I have addressed numbers 2, 3, and 5 from @wdduncan's list above.  For the rest, I'll defer to the group consensus and ask @cstoeckert for an opinion.

**Comment by wdduncan** [comment]
@nataled I've thinking about this past few days, and I don't have any real answer regarding `container amount measured by` vs. `container amount measured as`. We can discuss at tomorrow's RO meeting. But here are some random thoughts.

1. It would be nice if the label `contains amount` had a nice inverse. Not sure what this is though ... Perhaps `amount in container`?
2. Normally, I'd be okay with using the label `contains amount`. But, there has been recent discussion about including data properties in RO. Maybe I'm worrying to much, and we should just use the label as is. If a data property for amounts in containers is proposed, we can address it then. (i.e., for now we kick the can down the road).

In any case, let's focus on wrapping this up at the meeting. This issue doesn't need to hang around too long.

File: src/ontology/ro-edit.owl
Comment (wdduncan) [inline src/ontology/ro-edit.owl:569]: Please change IRI to `obo:RO_0017005`. I made a mistake when I assigned `obo:RO_0017002`.

Comment (jmwhorton) [reply src/ontology/ro-edit.owl]: RO_0017002 has been corrected to RO_0017005

**Comment by wdduncan** [comment]
Agreed on RO call to change label to `contains measured amount`.

File: src/ontology/imports/other_import.owl
Comment (anitacaron) [inline src/ontology/imports/other_import.owl:60]: There's an OBI import now. Please add the CURIE ids to the `src/ontology/imports/obi_terms.txt` and run `sh run.sh make imports/obi_import.owl`. Let me know if you need help, or I can do it.

**Review by anitacaron — CHANGES_REQUESTED** [review]

File: src/ontology/imports/other_import.owl
Comment (wdduncan) [reply src/ontology/imports/other_import.owl:60]: @jmwhorton have you had a chance to review @anitacaron requested change?

Comment (jmwhorton) [reply src/ontology/imports/other_import.owl:60]: Apologies, it's been a busy few weeks.  I should be able to take care of it this week.  Appreciate you checking in on it.

Comment (anitacaron) [reply src/ontology/imports/other_import.owl:60]: Please make sure to pull the changes in your local branch version. 😄

**Comment by wdduncan** [comment]
Fixes #712

File: src/ontology/imports/other_import.owl
Comment (jmwhorton) [reply src/ontology/imports/other_import.owl]: The obi terms have been moved, and the `make` command has been run.

**Comment by jmwhorton** [comment]
Added example of usage.  I believe all issues have been addressed, and this is ready to be reviewed again.

**Comment by wdduncan** [comment]
Thanks @jmwhorton  
Did you manually add the domain ([device](http://purl.obolibrary.org/obo/OBI_0000967)) and range ([measurement data item](http://purl.obolibrary.org/obo/IAO_0000109)] to `obi_import.owl` manually?

If so, I believe those terms should be added to the `obi_terms.txt` file, and the import should be rebuilt.  Otherwise, the changes you made to `obi_import.owl` will get overwritten by running ODK.

@matentzn @anitacaron can you confirm this?  I can't quite recall the ODK command to rebuild and OBI import.  I think it is:`sh run.sh make refresh-obi`. However, is it better to update the whole repo, i.e., `sh run.sh make update_repo`?

**Comment by wdduncan** [comment]
P.S. I just noticed @anitacaron comment. Not sure if it better to run `sh run.sh make refresh-obi` or `sh run.sh make imports/obi_import.owl`. I think they may do the same thing.

**Comment by jmwhorton** [comment]
For the sake of clarity I'll answer both the previous commends, @wdduncan.  I didn't add the imports to obi_import.owl manually.  I did run `make imports/obi_import.owl` 

However, do note that I added terms into other_import.owl manually (when I originally made the change, all of the imports were added there.  I manually added them to the .owl file since other_terms.txt was empty).

Happy to make any changes if I need to.

**Comment by wdduncan** [comment]
Thanks @jmwhorton !  
I believe the new terms should go in `obi_terms.txt`. Let's wait on @anitacaron or @matentzn to confirm.

**Comment by jmwhorton** [comment]
To be specific, the new terms that I left in `other_terms.owl` are IAO terms, not OBI terms.  (so I didn't want to put them in the OBI file)

**Comment by wdduncan** [comment]
Thanks. I didn't catch that. Ideally, we should create new IAO import, but measurement data item is in OBI ... What you think @anitacaron ?

File: src/ontology/imports/other_import.owl
Comment (anitacaron) [reply src/ontology/imports/other_import.owl]: Thank you, @jmwhorton!

**Comment by anitacaron** [comment]
You're right, @wdduncan. We need to create an IAO import and add the IAO term there instead of adding it to the `other_import.owl` file.

**Comment by wdduncan** [comment]
@anitacaron Can you modify the ODK file to create the IAO import? Also, is there documentation about the purpose of `other_import.owl`?

**Comment by ddooley** [comment]
One question - can the IAO terms be imported by way of COB?  I recall there was going to be some change made to COB import to enable 3rd party terms imported that way?

**Comment by wdduncan** [comment]
@anitacaron will be able to modify the ro-odk.yaml so that IAO is included as an import?  
Also, do you have any documentation on what `other_import.owl` is used for?

**Comment by anitacaron** [comment]
@wdduncan I think the `other_imports.owl` was created to have all imports, but it was manually updated. There's a comment in the file.

> Annotation(rdfs:comment "Ad-hoc additional imports, manually copied. This should be split into several dynamically imported sub-modules.")

Ideally, we should remove this file. Now it only imports a few terms. It would be good to review, update, and create a proper import in another issue. For example, instead of [disease (OGMS)](http://purl.obolibrary.org/obo/OGMS_0000031), use [disease (MONDO)](http://purl.obolibrary.org/obo/MONDO_0000001).

> Can the IAO terms be imported by way of COB?

Now that we have the COB import, you can import the COB terms like OBI. Add the terms in the `src/ontology/imports/cob_terms.txt` file and run `sh run.sh make imports/cob_import.owl`
I found [measurement datum - COB:0000121](https://www.ebi.ac.uk/ols4/ontologies/cob/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FCOB_0000121?lang=en) and [data item - IAO:0000027](https://www.ebi.ac.uk/ols4/ontologies/cob/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FIAO_0000027?lang=en). But only the `measurement datum` is used in the new RO property, so I think you only need to add this term to the `cob_terms.txt` file.

File: src/ontology/imports/other_import.owl
Comment (anitacaron) [inline src/ontology/imports/other_import.owl:75]: Please, remove this entry because it's already defined in the `core.owl` artefact.

File: src/ontology/imports/other_import.owl
Comment (anitacaron) [inline src/ontology/imports/other_import.owl:61]: Please, import the COB term [COB:0000121](https://www.ebi.ac.uk/ols4/ontologies/cob/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FCOB_0000121?lang=en) instead and remove this entry here.

File: src/ontology/imports/other_import.owl
Comment (anitacaron) [inline src/ontology/imports/other_import.owl:70]: Please, remove these additional classes related to the term above.

**Review by anitacaron — CHANGES_REQUESTED** [review]

File: src/ontology/imports/other_import.owl
Comment (jmwhorton) [reply src/ontology/imports/other_import.owl]: Removed

File: src/ontology/imports/other_import.owl
Comment (jmwhorton) [reply src/ontology/imports/other_import.owl]: Removed

**Comment by jmwhorton** [comment]
Requested changes have been made, @wdduncan

**Comment by jmwhorton** [comment]
@anitacaron 'dc' has been replaced with 'terms' now to fit with recent changes to master.

**Comment by wdduncan** [comment]
@anitacaron I don't understand which changes you requested. Have they been addressed?

**Comment by anitacaron** [comment]
Yes, they were addressed, and all conversations were resolved. I'm waiting for someone from the ontology to approve it, and I'll approve it after.

**Review by wdduncan — APPROVED** [review]

**Comment by wdduncan** [comment]
> Yes, they were addressed, and all conversations were resolved. I'm waiting for someone from the ontology to approve it, and I'll approve it after.

Ok. I approved.

**Comment by wdduncan** [comment]
Thanks @anitacaron !!!!

**Pull request merged**
